microservices-demo/kustomize/base/shippingservice.yaml

```diff
@@ -41,7 +41,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/shippingservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```

Comment:
I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/shippingservice:1.0.0
```